backend/src/controller/layers.rs

```diff
@@ -6,10 +6,7 @@ use actix_web::{
     HttpResponse, Result,
 };
 
-use crate::{
-    config::data::AppDataInner,
-    model::dto::{LayerSearchParameters, PageParameters},
-};
+use crate::{config::data::AppDataInner, model::dto::LayerSearchParameters};
 use crate::{model::dto::NewLayerDto, service::layer};
 
 /// Endpoint for searching layers.
```

Comment:
Documentation not updated (line below doesn't make sense if we use Vec instead of Page).